src/main/resources/application.yml

```diff
@@ -2,3 +2,12 @@ no-thanks:
   cookies:
     id: no-thanks-token
     name: no-thanks-name
+  gameParams:
+    coinsMap:
+      1: 11
+      2: 11
+      3: 11
+      4: 11
+      5: 11
+      6: 9
+      7: 7
```

Comment:
It's unused, isn't it? I see your `TODO`, but I don't see the point in adding this change now.